myriad-scheduler/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/recovery/MyriadFileSystemRMStateStore.java

```diff
@@ -58,6 +58,8 @@ public synchronized void initInternal(Configuration conf) throws Exception {
       //something is broken
       throw new RuntimeException("Could not find valid updateFile Method");
     }
+    
+    startInternal();
```

Comment:
Hi Everyone,

Addressed PR comments and did a bit more cleanup. Last unaddressed issue is
the unit test naming convention Darin mentioned. Please confirm if the
consensus is for me to change the names of my test methods to match
previously-written tests or vice versa.

Thanks

--John

On Sat, Jun 11, 2016 at 11:10 AM, John Yost <soozandjohnyost@gmail.com>
wrote:

> Hold on, having a git issue
>
> On Sat, Jun 11, 2016 at 11:03 AM, John Yost <soozandjohnyost@gmail.com>
> wrote:
>
>> Whoops, sorry about that, now it's gone
>>
>> --John
>>
>> On Sat, Jun 11, 2016 at 9:06 AM, Darin <notifications@github.com> wrote:
>>
>>> In
>>> myriad-scheduler/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/recovery/MyriadFileSystemRMStateStore.java
>>> <https://github.com/apache/incubator-myriad/pull/78#discussion_r66705718>
>>> :
>>>
>>> > @@ -58,6 +58,8 @@ public synchronized void initInternal(Configuration conf) throws Exception {
>>> >        //something is broken
>>> >        throw new RuntimeException("Could not find valid updateFile Method");
>>> >      }
>>> > +
>>> > +    startInternal();
>>>
>>> Seems to still be there
>>>
>>> —
>>> You are receiving this because you were mentioned.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/apache/incubator-myriad/pull/78/files/b85a73928ee07cc70a4c174eddffc25f5705b7f4#r66705718>,
>>> or mute the thread
>>> <https://github.com/notifications/unsubscribe/AKSRgR_NhsrisAGT1302pnK8KO2-4oQpks5qKrLwgaJpZM4IyN3c>
>>> .
>>>
>>
>>
>